src/picongpu/include/particles/ionization/byField/ADK/ADK.def

```diff
@@ -61,9 +83,13 @@ namespace ionization
      * \see speciesDefinition.param
      */
     template<typename T_DestSpecies>
-    struct ADK
+    struct ADKCircPol
     {
-        typedef ADK_Impl<T_DestSpecies> type;
+        /* Boolean value that results in an additional polarization factor in
+         * the ionization rate for linear polarization */
+        BOOST_STATIC_CONSTEXPR bool linPol = false;
+        typedef particles::ionization::AlgorithmADK<linPol> IonizationAlgorithm;
```

Comment:
Please give a struct with the float value of the polarization factor instead of a `bool`.